### ISSUE TITLE
`runjags` is a dependency, and it doesn't build on Windows

### ISSUE THREAD

**Issue by CGMossa** [ku-awdc/hexscape#2]
`{runjags}` is in `imports`: https://github.com/ku-awdc/hexscape/blob/fb5f67af649953958d225a04ca7bd68bb5a33d53/DESCRIPTION#L35

but it could just be in `Suggests`.

Build error:
```
* installing *source* package 'runjags' ...
** using staged installation
** libs
using C compiler: 'gcc.exe (GCC) 12.2.0'
using C++ compiler: 'G__~1.EXE (GCC) 12.2.0'
g++ -std=gnu++17  -I"C:/Users/minin/scoop/apps/r/current/include" -DNDEBUG -I"/c/progra~1/JAGS/JAGS-4.3.1/include" -D JAGS_MAJOR_ASSUMED=0 -D JAGS_MAJOR_FORCED=04    -I"c:/rtools43/x86_64-w64-mingw32.static.posix/include"     -O2 -Wall  -mfpmath=sse -msse2 -mstackrealign  -c distributions/jags/DFunction.cc -o distributions/jags/DFunction.o
In file included from distributions/jags/DFunction.h:13,
                 from distributions/jags/DFunction.cc:12:
distributions/jags/../../jagsversions.h:28:10: fatal error: version.h: No such file or directory
   28 | #include <version.h>
      |          ^~~~~~~~~~~
compilation terminated.
make: *** [C:/Users/minin/scoop/apps/r/current/etc/x64/Makeconf:270: distributions/jags/DFunction.o] Error 1
ERROR: compilation failed for package 'runjags'
* removing 'C:/Users/minin/scoop/persist/r/site-library/runjags'
Warning in install.packages :
  installation of package ‘runjags’ had non-zero exit status
ERROR: dependency 'runjags' is not available for package 'hexscape'
* removing 'C:/Users/minin/scoop/persist/r/site-library/hexscape'
Warning in install.packages :
  installation of package ‘hexscape’ had non-zero exit status
```

**Comment by CGMossa** [comment]
This is due to not having JAGS installed. Installation link: https://sourceforge.net/projects/mcmc-jags/files/latest/download

**Comment by mdenwood** [comment]
Fixed in commit 6dc7738a357bc913e497f3937b07d24e85dfee88
